### ISSUE TITLE
Background color in matplotlib - white vs. transparent

### ISSUE THREAD

**Issue by danielhuppmann** [IAMconsortium/pyam#426]
Glen Peters pointed out in [this tweet](https://twitter.com/Peters_Glen/status/1305764818633269248) that copy-pasting pyam/matplotlib figures into social media leads to suboptimal results. Any thoughts on whether to set the background color to white for the entire figure, or add an option to do this easily?

![image](https://user-images.githubusercontent.com/16931589/93197651-35c68800-f74c-11ea-9193-7b530d847aff.png)

**Comment by gidden** [comment]
I wonder if we could use a default theme, e.g., [seaborn's white grid](https://seaborn.pydata.org/tutorial/aesthetics.html#seaborn-figure-styles) that is more to our liking? Not sure if this would fix the background issue or not...

**Comment by jkikstra** [comment]
I agree that it would be odd to have white background within axes but transparent outside for default.
Might be useful in some cases, but generally non-transparent background is better i think.
